src/logic/operate.js

```diff
@@ -0,0 +1,57 @@
+import Big from "big-js";
+
+export default function operate(num1, num2, operation) {
+  switch (operation) {
+    case "+":
+      return add(num1, num2);
+    case "-":
+      return substract(num1, num2);
+    case "x":
+      return multiply(num1, num2);
+    case "/":
+      return divide(num1, num2);
```

Comment:
dont forget to handle division by ZERO when using this component

Reply:
thank you, will add that case!